ChopChop/ViewModels/Graph/SelectionHandler.swift

```diff
@@ -0,0 +1,26 @@
+import Combine
+import Foundation
+
+final class SelectionHandler<T: Node>: ObservableObject {
+    @Published private(set) var selectedNodeIds: [UUID] = []
```

Comment:
Are we allowing selection of multiple nodes?

Reply:
Not for now.